src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/b99fc951-3d63-47fb-a54c-40597b4f090d)

Reply:
methods has been divided into smaller methods, created constants for 'supply' and 'buy'";
[master 1344b28] methods has been divided into smaller methods, created constants for 'supply' and 'buy'
Thank you!